### DISCUSSION TITLE
pom文件中无法解析符号

### DISCUSSION THREAD

**Discussion by WZR8277** [zuihou/zuihou-admin-cloud#229] [02.提问]
请问我的pom文件报错，显示"无法解析符号sign"，这种错误如何解决呢？
![image](https://user-images.githubusercontent.com/68729543/204548512-a31708ed-8d0d-48f0-9297-b5e989c14537.png)

**Comment by zuihou** [comment]
查一下1.6版本是否支持sign参数

**Comment by WZR8277** [comment]
谢谢，参考https://blog.csdn.net/xiaolyuh123/article/details/107321245这个链接已经解决。
